website/docs/r/global_cluster_config.html.markdown

```diff
@@ -72,51 +74,6 @@ description: |-
 	}
 ```
 
-### Example Global cluster config
```

Comment:
this example must be a leftover, it's using a replicaset cluster and doesn't add any value or info over the previous example